resources/scponly_user.rb

```diff
@@ -82,6 +89,14 @@
       manage_home true
       shell '/usr/bin/scponly'
     end
+
+    group "scponly append #{new_resource.name}" do
+      group_name 'scponly'
+      append true
+      members new_resource.name
+      action :modify
+    end
+
```

Comment:
What you're doing here is replicating code between the two if blocks. I recommend you move this down to where the resource was initially. Why did you split it out like this?

Reply:
I was still encountering idempotency issues due to the `directory "#{altroot}/etc"` block (in my previous comment). In our meeting Friday, you recommended putting it the group resource above the if/else block, but the user is created within the if/else block.